pkg/operator/ceph/object/spec.go

```diff
@@ -34,7 +37,23 @@ import (
 )
 
 const (
-	livenessProbePath = "/swift/healthcheck"
+	livenessProbePath   = "/swift/healthcheck"
+	vaultBackendPath    = "VAULT_BACKEND_PATH"
+	vaultBackend        = "VAULT_BACKEND"
```

Comment:
use `secrets.VaultBackendKey` instead.

Reply:
I was not be to find this const in libopenstorage

Reply:
ditto